docs/config.md

```diff
@@ -158,4 +156,18 @@ chains.tekton.dev/transparency-upload: "true"
 > the path specified by `signers.kms.auth.token-dir`. 
 
 > [!IMPORTANT]
-> To project the latest token values without needing to recreate the pod, avoid using `subPath` in volume mount.
\ No newline at end of file
+> To project the latest token values without needing to recreate the pod, avoid using `subPath` in volume mount.
+
+## Namespaces Restrictions in Chains Controller
+This feature allows you to specify a list of namespaces for the controller to monitor, providing granular control over its operation. If no namespaces are specified, the controller defaults to monitoring all namespaces.
+
+Usage
```

Comment:
This line and the next are rendered as one line. Did you intend something like this instead:

```md
### Usage

To restrict...
```

That will render as:

> ### Usage
> 
> To restrict...

Or maybe just add an extra line break:

```md
Usage

To restrict...
```

which should render as:

> Usage
>
> To restrict...

Same for the other header-like line in this section.